bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -142,19 +146,34 @@ public Response getInterpreter(
     public Response interpret(
             @HeaderParam(HttpHeaders.ACCEPT_LANGUAGE) @Parameter(description = "language") @Nullable String language,
             @Parameter(description = "text to interpret", required = true) String text,
-            @PathParam("id") @Parameter(description = "interpreter id") String id) {
+            @PathParam("ids") @Parameter(description = "comma separated list of interpreter ids") String ids) {
         final Locale locale = localeService.getLocale(language);
-        HumanLanguageInterpreter hli = voiceManager.getHLI(id);
-        if (hli == null) {
+        Collection<HumanLanguageInterpreter> hlis = voiceManager.getHLIsByIds(ids);
+        if (hlis == null) {
             return JSONResponse.createErrorResponse(Status.NOT_FOUND, "No interpreter found");
         }
-
-        try {
-            hli.interpret(locale, text);
-            return Response.ok(null, MediaType.TEXT_PLAIN).build();
-        } catch (InterpretationException e) {
-            return JSONResponse.createErrorResponse(Status.BAD_REQUEST, e.getMessage());
+        String answer = null;
+        String error = null;
+        for (var interpreter : hlis) {
+            try {
+                answer = interpreter.interpret(locale, text);
+                logger.debug("Interpretation result: {}", answer);
+                error = null;
+                break;
+            } catch (InterpretationException e) {
+                logger.debug("Interpretation exception: {}", e.getMessage());
+                var msg = e.getMessage();
+                if (msg != null) {
+                    error = msg;
+                }
+            }
+        }
+        if (answer != null) {
+            return Response.ok(answer, MediaType.TEXT_PLAIN).build();
+        } else if (error != null) {
+            return JSONResponse.createErrorResponse(Status.BAD_REQUEST, error);
         }
+        return Response.ok("", MediaType.TEXT_PLAIN).build();
```

Comment:
I suggest rather:
```
        if (error != null) {
            return JSONResponse.createErrorResponse(Status.BAD_REQUEST, error);
        } else {
            return Response.ok(answer == null ? "" : answer, MediaType.TEXT_PLAIN).build();
        }
```
As the previous code was building the OK message with null, I am not sure we have to check if answer is null or not. Maybe this code is OK:
```
        if (error != null) {
            return JSONResponse.createErrorResponse(Status.BAD_REQUEST, error);
        } else {
            return Response.ok(answer, MediaType.TEXT_PLAIN).build();
        }
```